packages/itwinui-react/src/core/Tooltip/Tooltip.tsx

```diff
@@ -242,13 +266,15 @@ const useTooltip = (options: TooltipOptions = {}) => {
   );
 
   const floatingProps = React.useMemo(
-    () =>
-      interactions.getFloatingProps({
+    () => ({
+      ...interactions.getFloatingProps({
         hidden: !open,
         'aria-hidden': 'true',
         ...props,
         id,
       }),
+      popover: 'manual',
```

Comment:
Don't think this would matter, but would it be better to put `popover: 'manual'` inside `getFloatingProps`? Or maybe it's outside intentionally to force `popover: 'manual'`?

Reply:
I kept it outside so I don't have to deal with types. (`popover` types are not currently available)